File: Web Alex Roel/s7-JS-Estructuras-Array/break-continue.js
let c = 0;

while(c<=9){
    c++;
    
    if(c === 5){
        console.log('Termina el bucle');
        break;
    }
    if(c === 3){
        console.log('Salta a la siguiente linea');
        continue;
    }
    console.log(c);
}

File: Web Alex Roel/s33-Go Control/break-continue.go
package main

import "fmt"

func main() {
	for i := 0; i <= 10; i++ {

		if i == 5 {
			fmt.Println("Salta la iteracion")
			continue
		}
		if i == 8 {
			fmt.Println("Romper bucle")
			break
		}
		fmt.Println(i)
	}
}
